server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3082,6 +3082,7 @@ public TrafficTypeResponse createTrafficTypeResponse(PhysicalNetworkTrafficType
         PhysicalNetwork pnet = ApiDBUtils.findPhysicalNetworkById(result.getPhysicalNetworkId());
         if (pnet != null) {
             response.setPhysicalNetworkId(pnet.getUuid());
+            response.setIsolationMethods(pnet.getIsolationMethods());
```

Comment:
Worth testing if the serialised API response (json/xml), converts the string list as expected.

Reply:
`cmk` is able to display the array, I've put the output in the PR description if you want to take a look